app/views/posts/index.html.erb

```diff
@@ -1 +1,37 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<main>
+  <div class="list">
+      <div class="user">
+        <div class="user_details">
+          <div class="user_photo">
+            <%= image_tag("mybaby.png") %>
+          </div>
+          <div class="user_name">
+            <h2><%= @user.name %></h2>
+            <h4>Number of posts: <%= @user.post_counter %></h4>
+          </div>
+        </div>
+      </div>
+
+      <% @posts.each_with_index do |post, index| %>
+        <div class="user_posts">
+          <h3>Post #<%=index+1%>: <%= link_to "#{post.title}", user_post_path(@user.id, post.id) %></h3>
+          <p><%= post.text %></p>
+          <h4><span>Comment <%= post.comments_counter %></span> <span>|</span> <span>Likes <%= post.likes_counter %></span></h4>
+          <hr>
+          <div class="user_comments">
+            <% if post.most_recent_comments.length == 0 %>
+              <p>No comments for now. Be the first to comment</p>
+            <% else %>
+              <% post.most_recent_comments.each do |comment| %>
+                <p><%= comment.user.name%>: <%= comment.text%></p>
+              <%end%>
+            <% end %>
+          </div>
+        </div>
+      <% end %>
```

Comment:
- [ ] Upon accessing this page an error is produced, kindly make sure to fix this so all the pages are displaying the right information

![Screenshot_20230404_090059](https://user-images.githubusercontent.com/25227022/229742908-5ebfd0f7-708f-4abb-8600-bfec462d435b.png)